openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/roles/RolesPageTest.java

```diff
@@ -0,0 +1,136 @@
+package org.openmetadata.catalog.selenium.pages.roles;
+
+import com.github.javafaker.Faker;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.openmetadata.catalog.selenium.events.Events;
+import org.openmetadata.catalog.selenium.properties.Property;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.logging.Logger;
+
+@Order(19)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class RolesPageTest {
+
+  private static final Logger LOG = Logger.getLogger(RolesPageTest.class.getName());
+
+  static WebDriver webDriver;
+  static String url = Property.getInstance().getURL();
+  static Faker faker = new Faker();
+  static String roleDisplayName = faker.name().firstName();
+  static String enterDescription = "//div[@data-testid='enterDescription']/div/div[2]/div/div/div/div/div/div";
+  static Actions actions;
+  static WebDriverWait wait;
+  Integer waitTime = Property.getInstance().getSleepTime();
+
+  @BeforeEach
+  public void openMetadataWindow() {
```

Comment:
The BeforeEach and AfterEach can be moved to another class called `SeleniumTest`, so that it can be inherited in other classes like so `public class RolesPageTest extends SeleniumTest`

This will reduce code repetition across different test classes

Reply:
Yes, planning on moving that part of the code to a single class.

Reply:
We can so that in a different PR